app/src/main/java/com/keylesspalace/tusky/BottomSheetActivity.kt

```diff
@@ -203,7 +205,9 @@ fun looksLikeMastodonUrl(urlString: String): Boolean {
         path.matches("^/objects/[-a-f0-9]+$".toRegex()) ||
         path.matches("^/notes/[a-z0-9]+$".toRegex()) ||
         path.matches("^/display/[-a-f0-9]+$".toRegex()) ||
-        path.matches("^/profile/\\w+$".toRegex())
+        path.matches("^/profile/\\w+$".toRegex()) ||
+        path.matches("^/p/\\w+/\\d+$".toRegex()) ||
+        path.matches("^/\\w+$".toRegex())
```

Comment:
This is getting pretty close to "just try opening all links in Tusky first" :grinning:

Reply:
It will indeed led to more false positives, but there are still a lot of links that are not matched, like ones with queries, multiple path segments, special characters, etc

Reply:
I was only half-joking - maybe it would be cleaner to actually do this (guarded by a preference?)

Reply:
This is one of the things that should just work, without users thinking about it (and I don't know how to name that preference)